examples/hello-world/hello.js

```diff
@@ -11,18 +16,15 @@ async function main() {
       context.run("/bin/sh", ["-c", "date"]);
       const future_result = yield context.commit();
       const { results } = await future_result;
-      task.accept_result(results[results.length - 1])
+      task.accept_result(results[results.length - 1]);
     }
   }
 
-  await asyncWith(
-    new Executor({ task_package: package, budget: "1.0", subnet_tag: "devnet-beta" }),
-    async (executor) => {
-      for await (let completed of executor.submit(worker, tasks)) {
-        console.log(completed.result().stdout);
-      }
+  await asyncWith(new Executor({ task_package, budget: "1.0", subnet_tag: "devnet-beta" }), async (executor) => {
```

Comment:
actually, I'd recommend against this kind of change, in my opinion it decreases the code's readability...

Reply:
this kind of formatting has been corrected automatically by the `prettier` tool with default rules. This can be tweaked of course, but I would leave the default rules which are often used in the js community